Packs/LinuxEventsCollection/README.md

```diff
@@ -1,52 +1,75 @@
 # Linux Events Collection
-This pack includes XSIAM content.
 
-## Collect Events from Vendor
+## Collector Configuration
 
-In order to use the collector, you can use one of the following options to collect events from the vendor:
+You could use any one of the following options to collect Linux events into Cortex XSIAM:
+  - [Broker VM Syslog Applet](#broker-vm)
+  - [Filebeat XDRC (XDR Collector)](#xdrc-xdr-collector)
 
-- [Linux Events Collection](#linux-events-collection)
-  - [Collect Events from Vendor](#collect-events-from-vendor)
-    - [Broker VM](#broker-vm)
-    - [XDRC (XDR Collector)](#xdrc-xdr-collector)
-
-In either option, you will need to configure the vendor and product for this specific collector.
+After the Cortex XSIAM Collector starts ingesting logs from the configured paths on the Linux servers, you could query the collected logs under the *`linux_linux_raw`* dataset. 
```

Comment:
Recommend:
After the Cortex XSIAM collector starts ingesting logs from the Linux servers, you can query the logs under the *`linux_linux_raw`* dataset.

Reply:
Fixed:
https://github.com/demisto/content/pull/36882/commits/be339141a34bd645960ec4e017b1ab64e68a7424#diff-547b4347d8e4a399e431acab3ca503579d3d23c0f8854847ee85c97343769646R9